### PR TITLE
Allow multiple filters

### PR DESCRIPTION
**Pull request by ChaseIndustries**
We'll need to have multiple filters in the future- updating this package to allow that functionality

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+
             if(comments) {
-                if (filter) {
-                    for (var i = comments.length - 1; i >= 0; i--) {
-                        if (comments[i].indexOf(filter) >= 0) {
-                            comments.splice(i, 1);
-                        }
-                    }
+                if(filter){ 
+                  if (typeof filter == "string") {
```
Comment (bigtimebuddy) [inline index.js]: Please use strict comparison `typeof filter === "string"`

File: index.js [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
+                    comments = filterComments(filter, comments);
+                  } else if(typeof filter  == "array"){
```
Comment (bigtimebuddy) [inline index.js]: The standardize way to check for an array is `Array.isArray(filter)`

**Comment by ChaseIndustries** [comment]
Matt I knew you'd think of a better way, new approach committed

**Comment by ChaseIndustries** [comment]
OK all set!

**Pull request merged**